### PR TITLE
Responsive SVG art, and removed tutorial

### LINKED ISSUE #3544: Remove docusaurus tutorial panel
image

On the front page of our docusaurus app there are basically 2 things currently: 
1. The green tutorial button
2. The SVG art thingy

Your task is simple - remove the green thing! Make sure it all still works after you do so.

### How to run Docusaurus locally?
- `cd src/web/docusaurus`
- Might need to also do `pnpm install`
- `pnpm start`
- Your browser should automatically open `http://localhost:4631`

### LINKED ISSUE #3545: Rework SVG in Docusaurus home page to be responsive
In #3486, we got a beautiful looking art for the Telescope docs home page. There are a few things we would like to fix:

- Remove all white space,
- Resize or rework the svg to make the page shorter,

https://user-images.githubusercontent.com/58532267/164485931-d48b0ab0-5e25-4ba8-8d1d-c8b6adf215a5.mp4

_Originally posted by @TueeNguyen in https://github.com/Seneca-CDOT/telescope/pull/3486#pullrequestreview-948764864_

### PR DESCRIPTION
**Pull request by 416rehman**
## Issue This PR Addresses  
Fixes #3545, fixes #3544

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description
Made the SVG art responsive as requested in #3545 and removed the tutorial panel as requested in #3544

Testing steps from #3544 

## Screenshots
desktop screen
mobile screen

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [x] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/web/docusaurus/src/pages/index.js [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
+//         <p className="hero__subtitle">{siteConfig.tagline}</p>
+//         <div className={styles.buttons}>
+//           <Link className="button button--secondary button--lg" to="./overview">
+//             Docusaurus Tutorial - 5min ⏱️
+//           </Link>
+//         </div>
+//       </div>
+//     </header>
+//   );
+// }
```
Comment (JiaHua-Zou) [inline src/web/docusaurus/src/pages/index.js:24]: We should remove this block of code and the imports related to this.

**Review by JiaHua-Zou — CHANGES_REQUESTED** [review]

File: src/web/docusaurus/src/pages/index.js (hunk 1 above)
Comment (416rehman) [reply src/web/docusaurus/src/pages/index.js]: cf02c291232df64a61e3ead66b2dac1fe77b56b0 Done

File: src/web/docusaurus/src/pages/index.js [hunk 2]
```diff
@@ -1,36 +1,16 @@
 import React from 'react';
-import clsx from 'clsx';
 import Layout from '@theme/Layout';
-import Link from '@docusaurus/Link';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
-import styles from './index.module.css';
```
Comment (JiaHua-Zou) [inline src/web/docusaurus/src/pages/index.js:6]: the `CSS` file needs to be remove since it is not used anymore.

**Review by JiaHua-Zou — DISMISSED** [review]

**Review by Kevan-Y — APPROVED** [review]
LGTM, needs to squash those commit.

**Review by sirinoks — APPROVED** [review]

**Review by AmasiaNalbandian — APPROVED** [review]

**Comment by humphd** [comment]
@416rehman awesome, keep these fixes coming!

**Pull request merged**